security/tailscale/pkg-descr

```diff
@@ -6,6 +6,13 @@ https://tailscale.com/
 Plugin Changelog
 ================
 
+1.0_1
```

Comment:
while I don't mind the _1 choice we don't do hotfix changelogs here. Either move to 1.0 or 1.1 for clarity

"initial release" is hardly ever expandable, but also difficult to explain in full